superflore/generators/ebuild/ebuild.py

```diff
@@ -225,19 +225,28 @@ def get_ebuild_text(self, distributor, license_text):
         # CMAKE_BUILD_TYPE
         if self.name == "catkin":
             ret += "BUILD_BINARY=\"0\"\n"
+
         ret += "ROS_DISTRO=\"{0}\"\n".format(self.distro)
         ret += "ROS_PREFIX=\"opt/ros/${ROS_DISTRO}\"\n"
-
+        # TODO(allenh1): This is gross. No.
```

Comment:
I'm regularly guilty of this as well, but instead of saying "This is gross." A more useful XXX or TODO comment might give an example of when the code will fail

Reply:
It's just there so I fix it before this gets merged (I will not merge that into the codebase -- that's just awful).

Reply:
Actually, on second thought, I really don't think this is as awful as I initially thought it was.

The more robust solution is to create an `eclass` for this, but that's not really in the spirit of things if it's just for unpacking things. So I think I'll just leave a note that says "if you need to update this, then do so. Otherwise, this is sufficient."